### DISCUSSION TITLE
media_proxy redirects to missing.png

### DISCUSSION THREAD

**Discussion by FireFury0** [mastodon/mastodon#21800] [Q&A]
When trying to load a video from a toot, the browser requests something like: https://[fqdn]/media_proxy/[id]/small or https://[fqdn]/media_proxy/[id]/original

On my installation, this always produces a 302 redirect to https://[fqdn]/files/small/missing.png, which is a 404.

If I tcpdump port 3000 on the loopback interface, I see the requests to /media_proxy/[id]/original and the response redirecting to missing.png.  The logs show:
bundle[2160537]: [034def04-918e-4d2a-8cb0-396fe7707053] method=GET path=/media_proxy/[id]/original format=html controller=MediaProxyController action=show status=302 duration=728.85 view=0.00 db=32.17 location=https://[fqdn]/files/original/missing.png

This was broken under 3.5.3, and still broken after an upgrade to 4.0.2.  I'm not sure where to start with debugging it.

Thanks.

**Comment by FireFury0** [comment]
It turns out that this was caused by ffmpeg not being installed.  Unfortunately this error isn't logged anywhere by default.

**Comment by danorton** [comment]
I’m seeing this on a webp file, but I definitely have ffmpeg installed.
